lib/graphql-bookshelf-bridge/fetcher.js

```diff
@@ -63,7 +63,7 @@ export default class Fetcher {
     let relation = model.where(`${tableName}.${idColumn}`, id)
     if (filter) relation = filter(relation)
     if (args) {
-      const whitelist = mapKeys((k) => snakeCase(k), pick(['isPublic'], args))
+      const whitelist = mapKeys((k) => snakeCase(k), pick(['isPublic', 'group_data_type'], args))
```

Comment:
Didn't `group_data_type` go bye-bye?

Reply:
worried this might be needed by the migrations? going to leave until i remove group_data_type everywhere a bit later